src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -757,7 +812,14 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             )
             logger.warning(warn_msg)
 
-        config_dict = cls.load_config(cached_folder)
+        dduf_reader = None
+        if dduf:
+            zip_file_path = os.path.join(cached_folder, dduf)
+            dduf_reader = DDUFReader(zip_file_path)
+            # The reader contains already all the files needed, no need to check it again
+            cached_folder = ""
```

Comment:
See: https://github.com/huggingface/diffusers/pull/10037/files#r1870594275.

Reply:
WDYM ?

Reply:
I think it's resolved. So, disregard.